addon/mongocrypt.cc

```diff
@@ -577,6 +577,8 @@ MongoCrypt::MongoCrypt(const CallbackInfo& info) : ObjectWrap(info) {
 
     mongocrypt_setopt_retry_kms(mongo_crypt(), true);
 
+    mongocrypt_setopt_enable_multiple_collinfo(mongo_crypt());
```

Comment:
Note: The driver may need a way to know if this is set

Reply:
This shouldn't be necessary, only an updated libmongocrypt will request more than one collInfo

Reply:
Isn't the issue when an updated libmongocrypt interacts with a driver that isn't updated though?

Reply:
Yeah, unfortunately having moved the JS code for the bindings into the driver means that that's a case we now need to handle – this option would have to be passed from JS to the constructor instead of being set unconditionally